src/desktop_notifier/macos_support.py

```diff
@@ -42,6 +43,7 @@ def is_bundle() -> bool:
     :returns: Whether we are inside an app bundle.
     """
     main_bundle = NSBundle.mainBundle
+    logger.debug(f"main_bundle.bundleURL: {main_bundle.bundleURL}")
```

Comment:
Not sure if this is actually helpful. I assume most clients very deliberately set their bundle ID.

Reply:
fair / will remove but you'd be surprised at what can happen when you build a bundle with a couple of sub bundles

Reply:
I see. How about adding this information to the warning logs instead, i.e., when it becomes relevant? Something along the lines of
```
logger.warning("Cannot verify signature of bundle %s. SecStaticCodeCheckValidity() failed with OSStatus: %s", bundleName, status)
```

Reply:
good idea; done.